Tests/scripts/lock_cloud_machines.py

```diff
@@ -0,0 +1,359 @@
+    """
+    deletes a lock queue file
+    Args:
+        storage_bucket(google.cloud.storage.bucket.Bucket): google storage bucket where lock machine is stored.
+        lock_repository_name(str): the lock repository  name.
+        job_id(str): the job id to delete.
+    """
+    file_path = f'{lock_repository_name}/{QUEUE_REPO}/{job_id}'
+    blob = storage_bucket.blob(file_path)
+    try:
+        blob.delete()
+    except Exception as err:
+        logging.debug(f'when we try to delete a build_from_queue = {file_path}, we get an error: {str(err)}')
+        pass
+
+
+def remove_machine_lock_file(storage_bucket: any, lock_repository_name: str, machine_name: str, job_id: str):
+    """
+    deletes a lock machine file
+    Args:
+        storage_bucket(google.cloud.storage.bucket.Bucket): google storage bucket where lock machine is stored.
+        lock_repository_name(str) the lock repository name.
+        machine_name: the machine to delete
+        job_id(str): the job id to delete.
+    """
+    file_path = f'{lock_repository_name}/{machine_name}-lock-{job_id}'
+    blob = storage_bucket.blob(file_path)
+    try:
+        blob.delete()
+    except Exception as err:
+        logging.debug(f'when we try to delete a lock machine file = {file_path}, we get an error: {str(err)}')
+        pass
+
+
+def lock_machine(storage_bucket: any, lock_repository_name: str, machine_name: str, job_id: str):
+    """
+    create a lock machine file
+    Args:
+        storage_bucket(google.cloud.storage.bucket.Bucket): google storage bucket where lock machine is stored.
+        lock_repository_name(str) the lock repository name.
+        machine_name: the machine to lock.
+        job_id(str): the job id that locks.
+    """
+    blob = storage_bucket.blob(f'{lock_repository_name}/{MACHINES_LOCKS_REPO}/{machine_name}-lock-{job_id}')
+    blob.upload_from_string('')
+
+
+def adding_build_to_the_queue(storage_bucket: any, lock_repository_name: str, job_id: str):
+    """
+    create a lock machine file
+    Args:
+        storage_bucket(google.cloud.storage.bucket.Bucket): google storage bucket where lock machine is stored.
+        lock_repository_name(str): the lock repository name.
+        job_id(str): the job id to be added to the queue.
+    """
+    blob = storage_bucket.blob(f'{lock_repository_name}/{QUEUE_REPO}/{job_id}')
+    blob.upload_from_string('')
+
+
+def get_my_place_in_the_queue(storage_client: storage.Client, gcs_locks_path: str, job_id: str):
+    """
+    get the place in the queue for job-id by the time-created of lock-file time-created.
+    Args:
+        storage_client(storage.Client): The GCP storage client.
+        gcs_locks_path(str): the lock repository name.
+        job_id(str): the job id to check.
+
+    Returns: the place in the queue.
+
+    """
+    logging.debug('getting all builds in the queue')
+    builds_in_queue = get_queue_locks_details(storage_client=storage_client, bucket_name='xsoar-ci-artifacts',
```

Comment:
Please use as constant:
LOCKS_BUCKET = 'xsoar-ci-artifacts'
```suggestion
    builds_in_queue = get_queue_locks_details(storage_client=storage_client, bucket_name=LOCKS_BUCKET,
```